shark/shark_benchmark_runner.py

```diff
@@ -112,24 +113,31 @@ def benchmark_frontend(self, modelname):
 
     def benchmark_torch(self, modelname):
         import torch
-        import torch._dynamo as dynamo
         from tank.model_utils import get_torch_model
 
         if self.device == "cuda":
             torch.set_default_tensor_type(torch.cuda.FloatTensor)
             if self.enable_tf32:
-                torch.backends.cuda.matmul.allow_tf32 = True
+                print(
```

Comment:
No, because SHARK/IREE TensorFloat32 is triggered by the same flag, and is fully functional. This is only to avoid the PyTorch TF32 flag from segfaulting the entire pytest suite